ddtrace/contrib/starlette/__init__.py

```diff
@@ -0,0 +1,9 @@
+from ...utils.importlib import require_modules
```

Comment:
We want to add the docstring here for the integration. Redis is a good example `ddtrace/contrib/redis/__init__.py`

Also remember to a starlette entry to `docs/integrations.rst`. 🙂